### ISSUE TITLE
enable content type form to be split across tabs

### ISSUE THREAD

**Issue by webmasterkai** [cape-io/cape-editor#30]
Might want to have things like general profile on one tab, artwork on another and maybe statement/bio long text fields on another.

**Comment by bjornmeansbear** [comment]
Yes, this was nice the way we organized it last time — as it stands now you are presented with a giant wall of content and it isn't as easy to get through and figure out where/what to do.
